workflows/src/main/resources/migration/accounts-workflows-schema-migration-v1.0-h2.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="R3.Corda" id="accounts-migrations-v1.0" dbms="h2">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="account_to_state_refs"/>
+            </not>
+        </preConditions>
```

Comment:
That is actually a cool workaround to make scripts work that you add to CorDapps for H2...